docs/managing-e2e-tests.md

```diff
@@ -169,6 +140,39 @@ Note that test-infra requires [Bazel](https://bazel.build/).
 
 At this point you can commit your changes and send a GitHub PR against the test-infra repository.
 
+#### Email alerts
+
+Test job failures can trigger email alerts. This can be configured when a job is added in a testgrid dashboard.
+For example:
+
+```
+# kubeadm-kind tests
+  - name: kubeadm-kind-master
+    test_group_name: ci-kubernetes-e2e-kubeadm-kind-master
+    alert_options:
+      alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
+    alert_stale_results_hours: 24
+    num_failures_to_alert: 12 # Runs every 2h. Alert when it's been failing for 24 hours
+```
+
+- `alert_mail_to_addresses` should be set to `kubernetes-sig-cluster-lifecycle@googlegroups.com`
+in the case of SIG Cluster Lifecycle. This is a mailing list (Google Group) that will receive the alert.
+- `alert_stale_results_hours` means an alert will be sent in case the job is in a stale state and is not reporting new status
+after N hours. Usually a restart by a test-infra "on-call" operator is required in such cases.
+- `num_failures_to_alert` sets the N failed runs ("red runs") after which an alert will be sent.
+
```

Comment:
If possible add a not that kubeadm default are 24h for older release, 8h for master

Reply:
adding note.